.github/workflows/benchmark-command.yml

```diff
@@ -89,9 +89,15 @@ jobs:
           role-to-assume: arn:aws:iam::369469875935:role/AutoGluonCIBenchmarkConfig
           role-duration-seconds: 3600
           aws-region: us-east-1
+      - name: Extract Fork Info
+        id: parse_fork_info
+        run: |
+          IFS="|" read -r fork_name fork_branch <<< "${{ github.event.inputs.fork_info }}"
+          echo "FORK_NAME=$fork_name" >> $GITHUB_OUTPUT
+          echo "FORK_BRANCH=$fork_branch" >> $GITHUB_OUTPUT
       - name: Generate AMLB User Dir
         run: |
-          /bin/bash CI/bench/generate_amlb_user_dir.sh ${{ github.event.inputs.fork_name }} ${{ github.event.inputs.fork_branch }} ${{ github.sha }}
+          /bin/bash CI/bench/generate_amlb_user_dir.sh ${{ steps.parse_fork_info.outputs.FORK_NAME }} ${{ steps.parse_fork_info.outputs.FORK_BRANCH }} ${{ github.sha }} "" ${{ github.event.inputs.folds }}
```

Comment:
Thanks for the fix

Reply:
Sure! 👍 
This was the benchmarking fix which picked wrong datasets: https://github.com/autogluon/autogluon/pull/3608/files#diff-d7a950e181b92ab6db738323d4bf38363908a944efb0d226ef1aa097d7caff17